src/Sylius/Behat/Page/Shop/Account/Order/IndexPageInterface.php

```diff
@@ -29,4 +30,9 @@ public function countOrders();
      * @return bool
      */
     public function isOrderWithNumberInTheList($number);
+
+    /**
+     * @param OrderInterface $order
+     */
+    public function beAbleToChangePaymentMethodForOrder(OrderInterface $order);
```

Comment:
*Primo*: missing ``@return bool``
*Secundo*: what would you say for a more... *bool-returning* method name, like ``isItPossibleToChangePaymentMethodForOrder(...)`` or sth like this?